charts/finrem-ccd-definitions/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: Financial Remedy - CCD Definitions
-name: finrem-ccd-definitions
-version: 1.1.0
+name: finrem-ccd-definitions-1
+version: 1.2
```

Comment:
I think minor version upgrade should be enough for this.

Reply:
Well, in fact that's the first time ever it's fully functional :)

It's a big issue